src/unwind.cc

```diff
@@ -111,7 +111,7 @@ DDRes unwindstate__unwind(UnwindState *us) {
 void unwind_pid_free(UnwindState *us, pid_t pid) {
   us->dso_hdr.pid_free(pid);
```

Comment:
Seeing this makes me wonder if having a single mapping `pid -> {dso, symbols, dwfl}` instead would make more sense.